src/state/apis/nft/types.ts

```diff
@@ -24,13 +23,15 @@ export type NftCollectionType = {
 export type NftItem = {
   // the ID of the actual token in the collection, not a Zapper/Covalent/Opensea ID internal ID, and not an AssetId either
   id: string
+  // the actual CAIP-22/29 of the asset
+  assetId: AssetId
   name: string // the name for this specific ID, should always be defined
   // The spot price of the NFT in crypto, denominated in the native asset of the NFT chain
   price: string
   chainId: ChainId
   description: string
-  // Not normalized as collectionId, we can't join since we don't store this normalized *yet*
-  // Once we split nft/nftApi, this can be collectionId and then join'd with the collection data
+  // TODO(gomes): store the collection data into nft[collectionAssetId] and make this `collectionId`
```

Comment:
Tackle me in this PR

Reply:
Actually not for this PR, tackling this as we speak and not only storing this fully normalized will require a huge lift as far as plumbing this goes, but we'll also have to make sure to consolidate the metadata in the normalized `NftItem`s, as we currently rely on the collection meta from the `NftItem` itself (which may or may not contain enough data) or from the `CollectionType` (separately fetched collection data, which is the same shape, but gotten from a diff source).

Will plumb through and consolidate in a PR stacked on top of this guy